app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    mv, old_file_name, new_path_file_name = command.split()
+
+    path = os.path.dirname(new_path_file_name)
+
+    new_file_name = new_path_file_name.split("/")[-1]
```

Comment:
Use  os.path.split instead of splitting by /. It will make your code cross-platform
Also remember about os.path.join